images/linux/scripts/helpers/install.sh

```diff
@@ -23,6 +23,8 @@ download_with_retries() {
     i=20
     while [ $i -gt 0 ]; do
         ((i--))
+        echo "Verifying $URL HTTP status code..."
+        verify_http_status_code $URL
```

Comment:
I think we should add an additional condition to start download only if we have a successful http_code. In other case we can download broken page like https://google.com/linux and keep going.